tests/s3/test_object_tagging.py

```diff
@@ -1209,41 +1163,25 @@ def test_user_defined_metadata_2481(self):
     @CTFailOn(error_handler)
     def test_maximum_object_tags_2482(self):
         """Add maximum nos. of Object tags >100 using json file."""
-        self.log.info(
-            "Add maximum nos. of Object tags >100 using json file")
-        self.log.info(
-            "Creating a bucket with name %s",
-            self.bucket_name)
-        resp = self.s3_test_obj.create_bucket(
-            self.bucket_name)
+        self.log.info("Add maximum nos. of Object tags >100 using json file")
```

Comment:
Let's correct this to greater than 10 tags

Reply:
Statement is updated now.